spec/models/user_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  let(:user_1) do
+    User.create(
+      name:     'Joe',
+      email:    'joe@mail.com',
+      password: 'joe_pass',
+      password_confirmation: 'joe_pass'
+    )
+  end
+
+  it 'is valid with a name, email, and password' do
+    expect(user_1).to be_valid
+  end
+
+  it 'is invalid without a name' do
+    user = User.new(name: nil)
+    user.valid?
+    expect(user.errors[:name]).to include("can't be blank")
+  end
+
+  it 'is invalid without an email address' do
+    user = User.new(email: nil)
+    user.valid?
+    expect(user.errors[:email]).to include("can't be blank")
+  end
+
+  it 'is invalid with a duplicate email address' do
+    user_1
```

Comment:
why we need here `user_1` if it is not used inside this block.

Reply:
fix